config/secrets.json.example

```diff
@@ -46,14 +46,16 @@
       "to": "Aggie <aggie-admin@example.com>",
       "from": "Aggie <aggie@example.com>",
       "level": "error",
-      "silent": false
+      "silent": false,
+      "disabled": true,
     },
     "Slack": {
       "webhookUrl": "https://hooks.slack.com/services/XXXXXXXXX/YYYYYYYYY/ZZZZZZZZZZZZZZZZZZZZZZZZ",
       "channel": "#aggie-deployments",
       "username": "Aggie@ServerName",
       "level": "error",
-      "handleExceptions": true
+      "handleExceptions": true,
+      "disabled": true,
```

Comment:
Let's put `disabled` as the very first field so it's obvious (easy to overlook at the bottom)

Reply:
Done :+1: